src/cbc_sdk/platform/policies.py

```diff
@@ -582,6 +655,94 @@ def object_rules(self):
             self._object_rules_need_load = False
         return self._object_rules
 
+    @property
+    def object_rule_configs(self):
+        """
+        Returns a dictionary of rule configuration IDs and objects for this Policy.
+
+        Returns:
+            dict: A dictionary with rule configuration IDs as keys and PolicyRuleConfig objects as values.
+        """
+        if self._object_rule_configs_need_load:
+            cfgs = self._info.get("rule_configs", self._info.get("rapid_configs", []))
+            ruleconfigobjects = [PolicyRuleConfig._create_rule_config(self._cb, self, cfg) for cfg in cfgs]
+            self._object_rule_configs = dict([(rconf.id, rconf) for rconf in ruleconfigobjects])
+            self._object_rule_configs_need_load = False
+        return self._object_rule_configs
+
+    def valid_rule_configs(self):
+        """
+        Returns a dictionary identifying all valid rule configurations for this policy.
+
+        Returns:
+            dict: A dictionary mapping string ID values (UUIDs) to dicts containing entries for name, description,
+                  and category.
+        """
+        presentation = self._get_ruleconfig_presentation()
+        return {k: {'name': v['name'], 'description': v['description'], 'category': v['presentation']['category']}
+                for k, v in presentation.items()}
+
+    @classmethod
+    def get_ruleconfig_parameter_schema_directly(cls, cb, ruleconfig_id):
```

Comment:
I suppose it could be consolidated, but at the risk of making the code less clear. This function does not take a policy ID, whereas the `get_ruleconfig_parameter_schema` method does. Again, it was a matter of keeping the functions clearer in my own mind.

Reply:
Does it even make sense in the Policy object given the path does not need a policy id. Should it instead be added to the CBCloudAPI object which also enables caching external from each Policy resource individually

Reply:
The idea is that, when `get_ruleconfig_parameter_schema` is called for a policy, if we have that policy's rule config presentation on hand, we can use _it_ to build the necessary return value, and not have to call through to `get_ruleconfig_parameter_schema_directly` and incur an API roundtrip.

Reply:
Now part of `CBCloudAPI` under the name `get_policy_ruleconfig_parameter_schema`.